modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosControllerEpics.scala

```diff
@@ -316,6 +327,11 @@ object GmosControllerEpics extends GmosEncoders {
           // TODO these are not smart about setting them only if needed
           setROI(config.bi, config.roi)
 
+      private def nsParams(state: GmosNSEpicsState, config: NSConfig): List[IO[Unit]] =
+        List(
```

Comment:
This `List` is unnecessary now but we'll add more elements to the list when configuring for N&S